E/edfplusdcnv/build_tarballs.jl

```diff
@@ -0,0 +1,37 @@
+using BinaryBuilder
+
+# Collection of pre-build edfplusdcnv binaries
+name = "edfplusdcnv"
+edfplusdcnv_ver = "1.0.0"
+version = VersionNumber(edfplusdcnv_ver)
+
+sources = [
+    ArchiveSource("https://www.teuniz.net/edfplusd-converter/edfplusdcnv_100.tar.gz", "eeb6033a2f21dc052852e3dd779da7d368f65d88e45b7e40fa14f4bb514e62b9"; unpack_target = "x86_64-linux-gnu"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd ${WORKSPACE}/
+mkdir -p "${bindir}"
+cp edfplusdcnv_*/* "${bindir}"
```

Comment:
The path of the file you want to install is clearly wrong, but I'm away from computer, I don't know what's inside and can't check myself. In the meantime, suggesting a better way to install file (still with the wrong path, so it won't work anyway):
```suggestion
install -Dvm 755 edfplusdcnv_*/edfplusdcnv "${bindir}/edfplusdcnv"
```